konf.py

```diff
@@ -0,0 +1,139 @@
+#! /usr/bin/env python3
+
+import yaml
+import jinja2
+import argparse
+
+
+# Custom Jinja2 functions
+@jinja2.contextfunction
+def getSiteName(context, addStaginPrefix=False):
```

Comment:
In Python, function and variable names should be written in snake case:

> Function names should be lowercase, with words separated by underscores as necessary to improve readability.
> 
> Variable names follow the same convention as function names.
> 
> mixedCase is allowed only in contexts where that's already the prevailing style (e.g. threading.py), to retain backwards compatibility.

https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

(You'd do well to read [PEP8](https://www.python.org/dev/peps/pep-0008/) and [PEP257](https://www.python.org/dev/peps/pep-0257/) in full if you haven't already)

Reply:
Following on from this - we tend to use [black](https://pypi.org/project/black/) with `--line-length 79` to format all our code - did you run it over your code here?

It's not too hard to configure VSCode or any other editor to do this for you every time you save or whatever. Ask if you want help with that.

Reply:
There's also a `g` missing in `addStaginPrefix`

Reply:
I've create this issue - but don't worry about it for this PR, we can solve it down the line.

https://github.com/canonical-web-and-design/deployment-configs/issues/283

Reply:
Sorry, I changed it